### DISCUSSION TITLE
Post types

### DISCUSSION THREAD

**Discussion by GarthDB** [DesignSystemsEngineer/designsystemsengineer.github.io#35] [Ideas]
The easiest thing to start with on this website is to create post collections of markdown files.

Here are a couple of ideas I was thinking of:

* articles
* links to external articles
* links to external resources/tools for building/maintaining design systems
* job posts
* events/conferences
  * post-conference report

I'm starting this discussion to open it up for additional ideas and feedback.

**Comment by nklansford** [comment]
I like the idea of articles. It could also be interesting to play around with video as a format. It can make it easier to follow tutorials or make the content feel more personal. I know when I'm trying to figure out how to do something I get the most our of articles that have a video tutorial of someone getting it to work.

Other questions I've been thinking about:

-  How should this site be described by visitors? A community? A resource list? A collaborative blog?
-  What is the primary need the site should fill? So far I've been picking up on "developer resources for spinning up a design system from scratch" and "a community for developers working on design systems"
- What ways would we facilitate engagement? It could be as simple as reactions for posts or we could go deeper in the community aspect. Is this accompanied by some kind of regular meeting or is the community all done via the site?

**Reply by GarthDB** [reply]
@nklansford, I was thinking if we were getting enough content regularly, we could have a live stream where we could jump on and discuss the news. It could be similar to the way [Adafruit shows off new products](https://www.youtube.com/watch?v=RenOqCBZ1Z8).

We could then turn that into a podcast if people wanted it.

**Reply by GarthDB** [reply]
I do want to build a course for building a design system from start to finish, but it will probably start as a series of articles while I plan it out.

**Reply by nklansford** [reply]
I dig the idea of the livestream coupled with good content.

**Reply by GarthDB** [reply]
@nklansford we could schedule it on twitch or zoom (depending on how many want to attend) and divvy up topics to those wanting to participate as reporting updates. I think even just jumping on a call and talking about these starting discussions could be useful.

**Comment by zastrow** [comment]
I think this is an excellent starting point list. There are a lot of resources out there, but bringing them together as well as reviewing/comparing them seems valuable.

> What ways would we facilitate engagement? It could be as simple as reactions for posts or we could go deeper in the community aspect. Is this accompanied by some kind of regular meeting or is the community all done via the site?

This does have me thinking about the Future of Jamstack project. They did a series of live Twitch streams which acted mostly like a public broadcast of a meeting and included audience questions and feedback during the discussion. It doesn’t look like they have the records available on [Twitch](https://www.twitch.tv/thefutureofjamstack), but I attended one or two of them and found it to be a really interesting way to engage with a community.

**Reply by GarthDB** [reply]
I hadn't heard about that project, but it sounds very similar to what I was thinking about and mentioned above with @nklansford

**Comment by GarthDB** [comment]
Another thing I was thinking about was a glossary of terms and definitions. If it's a set of markdown files in the SSG it would be nice to accept prs with new terms/updates.

**Comment by somerandomdude** [comment]
Are there resources for engineers to better understand the skillset(s) required to work in design systems? That seems to be a common ask from engineers I talk to and I ain't got nothing to point them to. I think that could be incredibly valuable.

**Reply by kristoferjoseph** [reply]
Something like this?
https://sean.voisen.org/writing/design-engineering-resources
